frontend/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "proxy": "http://localhost/3000",
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "bootstrap": "4.0.0",
+    "bootstrap-social": "5.1.1",
+    "cross-fetch": "2.1.0",
+    "font-awesome": "4.7.0",
+    "prop-types": "15.6.0",
+    "react": "^16.13.1",
+    "react-animation-components": "3.0.0",
+    "react-dom": "^16.13.1",
+    "react-popper": "0.9.2",
+    "react-redux-form": "1.16.8",
+    "react-scripts": "3.4.1",
+    "react-transition-group": "2.3.0",
+    "reactstrap": "5.0.0",
+    "redux": "3.7.2"
```

Comment:
Why you have added redux?

Reply:
That's just to use the redux form components like control. Not for state management.